Hunting Queries/AWSCloudTrail/AWS_Abnormal_userdata_download_from_AWS_EC2

```diff
@@ -0,0 +1,25 @@
+id: 152440f1-c1c2-4cc5-9b82-555a130c59d2
+name: AWS_Abnormal_userdata_download_from_AWS_EC2
+description: |
+  Detects abnormal downloading of User Data associated with AWS EC2 instances. Asset management, discovery, Vulnerability scanners are expected false positives in this.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+tactics:
```

Comment:
I think Execution-T1204 or Discovery-T1580 looks more correct than exfiltration.. as with userdata attribute you can execute the code at instance start similar to Runcommand.